corehq/apps/motech/openmrs/concepts/sync.py

```diff
@@ -0,0 +1,112 @@
+import json
+import jsonobject
+from corehq.apps.motech.openmrs.concepts.models import OpenmrsConcept
+from corehq.apps.motech.openmrs.restclient.listapi import OpenmrsListApi
+
+
+def sync_concepts_from_openmrs(account):
+    requests = account.get_requests_object()
+    OpenmrsConcept.objects.filter(account=account).delete()
+    api = OpenmrsListApi(requests, 'concept')
+    answers_relationships = []
+    for concept in api.get_all():
+        concept = openmrs_concept_json_from_api_json(concept)
+        concept, answers = openmrs_concept_from_concept_json(account, concept)
+        if answers:
+            answers_relationships.append((concept, answers))
+        concept.save()
+
+    while answers_relationships:
```

Comment:
I don't know what OpenMRS concepts are. Can you explain what this loop is doing? it looks to me as though it just looks in the database to see if all the `answers` are the same as the ones you received from the API. Is that correct? I might just be a bit slow this afternoon.